base/applications/wordpad/print.c

```diff
@@ -381,10 +381,9 @@ static void paint_ruler(HWND hWnd, LONG EditLeftmost, BOOL NewMetrics)
     HDC hdcPrint = make_dc();
     RECT printRect = get_print_rect(hdcPrint);
     RECT drawRect;
-    HBRUSH hBrush = CreateSolidBrush(GetSysColor(COLOR_MENU));
 
     GetClientRect(hWnd, &drawRect);
-    FillRect(hdc, &drawRect, hBrush);
+    FillRect(hdc, &drawRect, GetSysColorBrush(COLOR_MENU));
```

Comment:
This is Wine code: https://github.com/reactos/reactos/blob/a17664cec2ca0f98857e47b05d11cf5992096206/media/doc/WINESYNC.txt#L252

Reply:
Yeah, that's what I wrote.